analytics/Dockerfile

```diff
@@ -0,0 +1,125 @@
+# Use the official python3 image based on Debian 11 "Bullseye".
+# https://hub.docker.com/_/python
+
+# The build stage that will be used to deploy to the various environments
+# needs to be called `release` in order to integrate with the repo's
+# top-level Makefile
+FROM python:3-slim AS base
+
+# Install poetry, the package manager.
+# https://python-poetry.org
+RUN pip install --no-cache-dir poetry --upgrade
+
+RUN apt-get update \
+  # Install security updates
+  # https://pythonspeed.com/articles/security-updates-in-docker/
+  && apt-get upgrade --yes \
+  && apt-get install --no-install-recommends --yes \
+  build-essential \
+  libpq-dev \
+  postgresql \
+  wget \
+  # Reduce the image size by clear apt cached lists
+  # Complies with https://github.com/codacy/codacy-hadolint/blob/master/codacy-hadolint/docs/description/DL3009.md
+  && rm -fr /var/lib/apt/lists/* \
+  && rm /etc/ssl/private/ssl-cert-snakeoil.key
+
+# Install gh CLI
+# docs: https://github.com/cli/cli/blob/trunk/docs/install_linux.md
+RUN mkdir -p -m 755 /etc/apt/keyrings && wget -qO- https://cli.github.com/packages/githubcli-archive-keyring.gpg | tee /etc/apt/keyrings/githubcli-archive-keyring.gpg > /dev/null \
+  && chmod go+r /etc/apt/keyrings/githubcli-archive-keyring.gpg \
+  && echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | tee /etc/apt/sources.list.d/github-cli.list > /dev/null \
+  && apt update \
+  && apt install gh -y \
+  && gh --version
+
+ARG RUN_UID
+ARG RUN_USER
+
+# The following logic creates the RUN_USER home directory and the directory where
+# we will be storing the application in the image. This runs when the user is not root
+RUN : "${RUN_USER:?RUN_USER and RUN_UID need to be set and non-empty.}" && \
+  [ "${RUN_USER}" = "root" ] || \
+  (useradd --create-home --create --user-group --home "/home/${RUN_USER}" --uid ${RUN_UID} "${RUN_USER}" \
+  && mkdir /analytics \
+  && chown -R ${RUN_UID} "/home/${RUN_USER}" /analytics)
+
+# Set PYTHONPATH so that the tests can find the source code.
+ENV PYTHONPATH="/analytics/src/:$PYTHONPATH"
```

Comment:
I had to add this line

Reply:
Non-blocking request, but can we look into this further? If we're installing the `analytics/` package in the docker container correctly, I don't _think_ we should need to set the `PYTHONPATH` directly, see this [stack overflow post for more info](https://stackoverflow.com/a/70388156/7338319)

Reply:
Thanks for the feedback, but I'm not sure how valuable that would be right now, given that large set of work on my plate. I will make a ticket.

Reply:
Here's the ticket: https://github.com/HHS/simpler-grants-gov/issues/1565

Reply:
Yeah, I'm not super worried about it, mainly just want to flag it in case it resurfaces in some other error.